app/models/hyku_addons/csv_entry.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+module HykuAddons
+  class CsvEntry < Bulkrax::CsvEntry
+    def self.matcher_class
+      HykuAddons::CsvMatcher
+    end
+
+    # Override to add date and json field handling
+    def build_metadata
+      raise StandardError, 'Record not found' if record.nil?
+
+      raise StandardError, "Missing required elements, required elements are: #{importerexporter.parser.required_elements.join(', ')}" unless importerexporter.parser.required_elements?(record.keys)
```

Comment:
Because this line is so long, could you wrap the exception in a condition block rather than having it inline?

Reply:
I believe this is actually this way upstream and rubocop doesn't complain so :man_shrugging:.